Bio/Align/__init__.py

```diff
@@ -3317,6 +3321,75 @@ def map(self, alignment):
         alignment = Alignment(sequences, coordinates)
         return alignment
 
+    def mapall(self, alignments):
```

Comment:
Would ``map_all`` be OK or not much of an improvement?

Reply:
I am fine either way.